vendor/bat-native-ads/src/bat/ads/internal/conversions/conversions.cc

```diff
@@ -281,98 +275,109 @@ void Conversions::AddItemToQueue(const AdEventInfo& ad_event) {
     BLOG(6, "Successfully logged conversion event");
   });
 
-  ConversionQueueItemInfo queue_item;
-
-  const uint64_t rand_delay = brave_base::random::Geometric(
+  ConversionQueueItemInfo conversion_queue_item;
+  conversion_queue_item.campaign_id = ad_event.campaign_id;
+  conversion_queue_item.creative_set_id = ad_event.creative_set_id;
+  conversion_queue_item.creative_instance_id = ad_event.creative_instance_id;
+  conversion_queue_item.advertiser_id = ad_event.advertiser_id;
+  const int64_t rand_delay = brave_base::random::Geometric(
       g_is_debug ? kDebugConvertAfterSeconds : kConvertAfterSeconds);
```

Comment:
should we make this configurable via feature/variations?

Reply:
Agreed. However I have raised https://github.com/brave/brave-browser/issues/14215 so this can be prioritized at a later date. Thanks